source/_docs/solr.md

```diff
@@ -14,16 +14,15 @@ Apache Solr is a system for indexing and searching site content. Pantheon provid
 ## Which Plans Can Use Solr?
 
 All plans except for a Personal plan can use Solr. Solr is available to Sandbox plans for developmental purposes, but Solr will not be available going live on a Personal plan.
-
-| Plans        | Supported
-| ------------- |:-------------:|
-| Sandbox      | **Yes, for all environments** |
-| Personal      | No      |
-| Professional | **Yes, for all environments**      |
-| Business | **Yes, for all environments**      |
-| Elite | **Yes, for all environments**      |
-
-
+
+| Plans         | Supported <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-title="Environments" data-content="Solr is enabled for all environments"><em class="fa fa-info-circle"></em></a> |
```

Comment:
Let's be more specific in the second th - maybe **Solr Support** instead of Supported (keep the tooltip though)